providers/amplitude.go

```diff
@@ -0,0 +1,24 @@
+package providers
+
+const Amplitude Provider = "amplitude"
+
+func init() {
+	// Amplitude Support Configuration
+	SetInfo(Amplitude, ProviderInfo{
```

Comment:
Can we add in comment which API's are the ones that are supported with basic auth? Once comment added, feel free to merge.